libraries/AR_WPNav/AR_WPNav.cpp

```diff
@@ -216,7 +216,7 @@ bool AR_WPNav::set_desired_location(const struct Location& destination, float ne
         } else {
             // calculate maximum speed that keeps overshoot within bounds
             const float radius_m = fabsf(_overshoot / (cosf(radians(turn_angle_cd * 0.01f)) - 1.0f));
-            _desired_speed_final = MIN(_desired_speed, safe_sqrt(_turn_max_mss * radius_m));
+            _desired_speed_final = MIN(_desired_speed, safe_sqrt(_atc.get_g_limit_lateral_accel() * radius_m));
```

Comment:
as mentioned on the call let's try and keep the parameter name, the variable and the accessor all using consistent names.

Reply:
changed to `get_turn_lat_accel_max`